pkg/components/components.go

```diff
@@ -612,6 +612,11 @@ func GetClusterPermissions() []rbacv1.PolicyRule {
 			Resources: stringListToSlice("alertmanagers", "alertmanagers/api"),
 			Verbs:     stringListToSlice("get", "list", "create", "delete"),
 		},
+		{
+			APIGroups: stringListToSlice(""),
+			Resources: stringListToSlice("secrets"),
+			Verbs:     stringListToSlice("get", "list", "create", "update", "watch"),
+		},
```

Comment:
please add the missing permissions to the RBAC object in line 516, instead of this new object.

Reply:
done